local-apis/README.md

```diff
@@ -0,0 +1,3 @@
+# local-apis
+
+Stats directory is generated from a part of k8s.io/kubernetes.
```

Comment:
We might mention:
- what update.sh is for
- license term
  - Yes it's declared in types.go itself, but it's more kind.